### PR TITLE
IMSC 1.1 CR1

### PR DESCRIPTION
**Pull request by palemieux**
_DO NOT MERGE_

Prepare IMSC 1.1 for CR

### CODE REVIEW

**Comment by palemieux** [comment]
Built at https://rawgit.com/w3c/imsc/IMSC1.1-CR/imsc1/spec/ttml-ww-profiles.html

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -91,7 +92,17 @@
     <p>Both profiles are based on [[SUBM]].</p>
   </section>
 
-  <section id='sotd'></section>
+  <section id='sotd'>
+    <p>For this specification to exit the CR stage, at least 2 independent implementations of every feature defined in this
+    specification but not already present in [[TTML2]] need to be documented in the implementation report. The Working Group does
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: Needs to reference IMSC1.0.1 not TTML2 I think.

Comment (palemieux) [reply imsc1/spec/ttml-ww-profiles.html]: This matches the exit criteria for IMSC1, which only tested those features not already specified in TTML1.

Comment (nigelmegitt) [reply imsc1/spec/ttml-ww-profiles.html]: Ah yes, we need to say "not already in TTML2 and not already in IMSC1.0.1" to cover ourselves in the case of any new extensions in 1.1 (which may be an empty set).

**Pull request merged**